.github/workflows/changed.yaml

```diff
@@ -0,0 +1,158 @@
+name: Deploy references.toml
+on:
+  push:
+#    branches:
```

Comment:
Accidentally commented out?

Reply:
Intentionally; I needed to test the CI so I removed these limitations so it's triggered on every push. Will uncomment once it's the PR is finalised